### DISCUSSION TITLE
Missing TextureSet Files

### DISCUSSION THREAD

**Discussion by Noggog** [Mutagen-Modding/Mutagen.Bethesda.Analyzers#91] [Topic Codes]
# Context
`TextureSet.`
- `Diffuse`
- `NormalOrGloss`
- `EnvironmentMaskOrSubsurfaceTint`
- `GlowOrDetailMap`
- `Height`
- `Environment`
- `Multilayer`
- `BacklightMaskOrSpecular`

`TXST/`
- `TX00`
- `TX01`
- `TX02`
- `TX03`
- `TX04`
- `TX05`
- `TX06`
- `TX07`

# Description
Has a model listed, but the asset cannot be found in the load order

# Effects
purple texture in game

**Comment by erri120** [comment]
![2021 09 25-12 24 29  SkyrimSE_hrWCcqoN2T](https://user-images.githubusercontent.com/16577545/134859202-2782c2f8-6d23-4825-929d-f6a09fd094b3.png)
